packages/quantic/cypress/e2e/facets-2/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -209,6 +210,41 @@ describe('quantic-breadcrumb-manager', () => {
         });
       });
 
+      describe('when facet values are displayed as checkboxes', () => {
+        it('should properly display the breadcrumb values', () => {
+          visitBreadcrumbManager({
+            useCase: param.useCase,
+            displayFacetValuesAs: 'checkbox',
+          });
+
+          scope('when selecting values in facet', () => {
+            cy.then(Actions.facet.selectFirstLinkValue)
+              .wait(InterceptAliases.UA.Facet.Select)
+              .then((interception) => {
+                Expect.facetBreadcrumb.firstBreadcrumbValueLabelContains(
+                  interception.request.body.customData.facetValue
+                );
+              });
+            Expect.facetBreadcrumb.displayBreadcrumb(true);
+            Expect.facetBreadcrumb.labelContains('File Type');
+            Expect.facetBreadcrumb.displayValues(true);
+            Expect.facetBreadcrumb.numberOfValues(1);
```

Comment:
Would it make sense to make a function with these 4 expects if it's repeated multiple times?

Reply:
No not really, we are not really repeating the exact same thing, the expects are called with different parameters each time.